.ci.yaml

```diff
@@ -323,6 +323,25 @@ targets:
       - bin/
       - .ci.yaml
 
+  - name: Linux engine_dependency_proxy_test
+    recipe: devicelab/devicelab_drone
```

Comment:
Can you append a comment: `# New target: https://github.com/flutter/flutter/pull/115511`?

Reply:
done

Reply:
Nit: comment explaining `bringup: true` is preferred to append after the `bringup` entry.